release-team/role-handbooks/release-team-lead/README.md

```diff
@@ -296,6 +296,13 @@ Code Freeze will typically fall around Weeks 8 or 9 depending on the length or r
 ### Week 13
 
 - Release Retrospective participation
+  - Make sure to confirm that the Emeritus Adviser can serve as facilitator. If Emeritus Adviser is unavailable then defer the responsability as appropiate
```

Comment:
Small typo here: "responsibility"

Otherwise lgtm

Reply:
Fixed! Thank you @tpepper